### DISCUSSION TITLE
Prevent Submitting Deploy Command Without Flag Dependencies

### DISCUSSION THREAD

**Discussion by jmcnaughton-isuf** [forcedotcom/cli#2924] [Ideas]
Specifically, when submitting this command sf project deploy start --metadata ApexClass:MyApexClass -t MyApexClassTest.

The cli will allow you to submit this command, however, it will fail because the proper flag dependency "-l RunSpecifiedTests" is not included in the command.

**Comment by VivekMChawla** [comment]
@jmcnaughton-isuf - Thanks for posting your idea.

Could you clarify the behavior you'd expect from the CLI in this situation?
